### DISCUSSION TITLE
GoCD pipeline import

### DISCUSSION THREAD

**Discussion by egeland** [github/gh-actions-importer#248] [Feature requests] [answered]
Will GoCD pipelines be available to import into GH Actions using the importer?

**Comment by ethanis** [comment, marked as answer]
@egeland  thanks for your interest in GitHub Actions Importer! At this time, we don't have plans to support migrating GoCD pipelines to Actions automatically with the product. That said, we continually revise the product roadmap and will consider GoCD in the future.
